presto-native-execution/presto_cpp/main/common/tests/ConfigTest.cpp

```diff
@@ -255,4 +256,25 @@ TEST_F(ConfigTest, optionalNodeId) {
   EXPECT_EQ(nodeId, config.nodeId());
 }
 
+TEST_F(ConfigTest, readConfigEnvVarTest) {
+  setUpConfigFilePath();
+  std::string ENV_VAR = "PRESTO_READ_CONFIG_TEST_VAR";
```

Comment:
`std::string_view kEnvVar`

Reply:
This one needs to stay a string. We need the c_str() function that provides a null terminated `char*` . string_view does not provide this.

Reply:
This doesn't work because we need a null-terminated char* that string_view does not provide.